src/illuminatio/collect_results.py

```diff
@@ -14,81 +18,106 @@
 @click.option('--delimiter', default=';')
 @click.option('--quotechar', default='"')
 def cli(directory, outfile, delimiter, quotechar, performance):
```

Comment:
How can I use this command ? The illuminatio command doesn't provide this functionality:

```bash
illuminatio --help       
Usage: illuminatio [OPTIONS] COMMAND1 [ARGS]... [COMMAND2 [ARGS]...]...

Options:
  -v, --verbosity LVL  Either CRITICAL, ERROR, WARNING, INFO or DEBUG
  --incluster
  --help               Show this message and exit.

Commands:
  clean
  run
```

IMHO if we want to keep this (hidden?) feature it should be a subcommand of illuminatio e.g. `illuminatio results`

Reply:
It seems like this is an "artifact" from @maxbischoff